app/controllers/api/v0/highlights_controller.rb

```diff
@@ -101,6 +101,56 @@ def index
     render json: response_binding, status: :ok
   end
 
+  swagger_path '/highlights/{id}' do
+    operation :put do
+      key :summary, 'Update a highlight'
+      key :description, 'Update a highlight'
+      key :operationId, 'updateHighlight'
+      key :produces, [
+        'application/json'
+      ]
+      key :tags, [
+        'Highlights'
+      ]
+      parameter do
+        key :name, :id
+        key :in, :path
+        key :description, 'ID of the highlight to update'
+        key :required, true
+        key :type, :string
+        key :format, 'uuid'
+      end
+      parameter do
+        key :name, :highlight
+        key :in, :body
+        key :description, 'The highlight updates'
+        key :required, true
+        schema do
+          key :'$ref', :HighlightUpdate
+        end
+      end
+      response 200 do
+        key :description, 'Success.  Returns the updated highlight.'
+        schema do
+          key :'$ref', :Highlight
+        end
+      end
+      extend Api::V0::SwaggerResponses::AuthenticationError
+      extend Api::V0::SwaggerResponses::UnprocessableEntityError
+      extend Api::V0::SwaggerResponses::ServerError
+    end
+  end
+
+  def update
+    inbound_binding, error = bind(params.require(:highlight), Api::V0::Bindings::HighlightUpdate)
```

Comment:
do we want to rails log that an update happened?  anything that records what happened?

Reply:
We'll eventually be sending events to Kafka.  Will be more useful there than in the log.